src/families/cosmos/mock.js

```diff
@@ -13,7 +18,9 @@ const { validators } = preloadedData;
 function setCosmosResources(
```

Comment:
it's better using one object arg when there're more than two arguments especially, some of them are optional